.env.example

```diff
@@ -1,2 +1,13 @@
 API_URL=http://localhost:8080/api/
-IMAGE_NAME="frontend_app"
\ No newline at end of file
+IMAGE_NAME="frontend_app"
+
+HOMEPAGE_URL=https://fqq3vg4qpt.eu-west-1.awsapprunner.com/
+
+HOMEPAGE_URL_LOCAL=http://localhost:3000/
```

Comment:
1. You need to remove HOMEPAGE_URL_LOCAL
2. Then change HOMEPAGE_URL so it equals http://localhost:3000/